[No-tix] update global control group warning about content cards (#9954)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/testing/global_control_group.md

With the Global Control Group, you can set a percentage of all users as a control group. When saved, users in the group will not receive any campaigns or Canvases. 

> Your Global Control Group applies to all channels, campaigns, and Canvases, except for [API campaigns]({{site.baseurl}}/api/api_campaigns#api-campaigns). This means that users in your control group will still receive API campaigns.
> 
> However, this API-Triggered exception does not apply to Content Cards. If you are using an API-Triggered Content Card campaign, users in your control group will not receive them.

Reviewer: ```suggestion
{% alert important %}
Your Global Control Group applies to all channels, campaigns, and Canvases, except for [API campaigns]({{site.baseurl}}/api/api_campaigns#api-campaigns). This means users in your control group will still receive API campaigns. However, this exception doesn't apply to Content Cards. If you're using an API-triggered Content Card campaign, users in your control group won't receive them.
{% endalert %}
```
